src/test/java/edu/cornell/kfs/sys/web/mock/MockRemoteServerExtension.java

```diff
@@ -0,0 +1,75 @@
+package edu.cornell.kfs.sys.web.mock;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpHost;
+import org.apache.http.localserver.LocalServerTestBase;
+
+import edu.cornell.kfs.sys.web.mock.MockServiceEndpointBase;
```

Comment:
can this import be removed, it doesn't seem to be used?

Reply:
Good catch, I'll remove this import.